projects/swimlane/ngx-charts/src/lib/area-chart/area-chart-normalized.component.ts

```diff
@@ -448,17 +445,17 @@ export class AreaChartNormalizedComponent extends BaseChartComponent {
     return opts;
   }
 
-  updateYAxisWidth({ width }): void {
+  updateYAxisWidth({ width }: { width: number }): void {
```

Comment:
These look odd to me... are we passing some object that includes `width`?

Reply:
Yeah, it seems like it.